Core.ApplicationServices/SystemUsage/Write/ItSystemUsageWriteService.cs

```diff
@@ -257,6 +258,26 @@ private Maybe<OperationError> UpdateRegisteredDataCategories(ItSystemUsage syste
                 .MatchFailure();
         }
 
+        private static Maybe<OperationError> UpdatePersonalDataOptions(ItSystemUsage systemUsage, Maybe<IEnumerable<GDPRPersonalDataOption>> personalDataOptions)
+        {
+            var newOptions = personalDataOptions.GetValueOrFallback(new List<GDPRPersonalDataOption>()).ToList();
```

Comment:
us the ComputeDelta helper in stead of repeating your own delta logic here

Reply:
A good reason is that you forget to remove before you add... so if you call this twice with the same data it will fail on the first call to Add